### ISSUE TITLE
Broken 'View in Online Vega Editor' link in an example

### ISSUE THREAD

**Issue by avatorl** [vega/vega#3860]
At https://vega.github.io/vega/examples/zoomable-circle-packing/ link [View in Online Vega Editor](https://vega.github.io/editor/#/examples/vega/zoomable-circle-packing) opens HTML page 404 in Vega Editor, instead of JSON code of the example.

![image](https://github.com/vega/vega/assets/59934292/946db765-eb0d-43a7-8817-2b3d3bfe8d64)

@Giammaria

**Comment by Giammaria** [comment]
Yes, thanks. @domoritz pointed out The editor needs to be updated.
